docs/medical-api/handling-data/webhooks.mdx

```diff
@@ -366,6 +368,104 @@ The format follows:
+update and provide relevant information.
+
+You can expect to get one or two messages of this type for each Bulk Patient Create job, with the status
+being the main difference between each:
+
+- `waiting`: initial state, waiting for the CSV file;
+- `processing`: the CSV file has been uploaded and is being processed;
+- `completed`: the bulk import job is completed;
+- `failed`: the bulk import job failed (likely due to an uploaded file with invalid format).
+
+Here is an example payload:
+
+```json
+{
+  "meta": {
+    "messageId": "00000000-00000000-00000000-00000000",
+    "when": "2024-12-30T05:05:12.215Z",
+    "type": "medical.bulk-patient-create"
+  },
+  "job": {
+    "jobId": "00000000-00000000-00000000-00000000",
+    "status": "completed",
+    "result": "<presigned-download-url>"
+  }
+}
+```
+
+The format follows:
+
+<ResponseField name="meta" required>
+  Metadata about the message. The full format is described
+  [here](/medical-api/getting-started/webhooks#meta-data).
+  <Expandable title="meta type property details">
+    <ResponseField name="type" type="string" required>
+      For Bulk Patient Create this will always be
+      `medical.bulk-patient-create`.
+    </ResponseField>
+  </Expandable>
+</ResponseField>
+
+<ResponseField name="job" type="BulkPatientCreateJob" required>
+  The information about the Bulk Patient Create job.
+
+  <Expandable title="BulkPatientCreateJob properties">
+    <ResponseField name="jobId" type="string" required>
+      The ID that represents this bulk patient create job.
+    </ResponseField>
+  
+    <ResponseField name="status" type="string" required>
+      The status of the job. One of `processing`, `failed`, or `completed`.
+      See all the possible values
+      [here](/medical-api/api-reference/patient/bulk-create-patient#bulk-patient-create-status).
+    </ResponseField>
+
+    <ResponseField name="reason" type="string">
+      The reason for a failed job. Only present if the status
+      is `failed`.
+    </ResponseField>
+
+    <ResponseField name="result" type="string">
+      The URL to download the CSV file containing the result of
+      the Bulk Patient Create job. It's valid for 5 minutes.
+    </ResponseField>
+
+  </Expandable>
+</ResponseField>
+
+#### Result CSV File
+
+The message `medical.bulk-patient-create` with status `completed` includes a URL in the `result`
+field, which you can use to download the result of the Bulk Patient Create job.
+
+The file is based on the same format as the
+[input CSV](/medical-api/api-reference/patient/bulk-create-patient#input-csv-file-format), with the
+following additional columns:
+
+<ParamField body="metriportId" type="string">
+  The ID of the Patient on Metriport - only provided if the
+  Patiend was created successfully.
+</ParamField>
+<ParamField body="status" type="string" required>
+  Whether the respective demographics was accepted and used
+  to create a Patient. Can be one of `succeeded` or
+  `failed`.
```

Comment:
1. imo should be 'success' or 'failure' tbh. These things should usually just be present / infinitive tense, not past.
2. As a rule i think single short sentence of what the field is comes first always in docs. Additional details come after. Makes it much easier to rapidly skim the docs imo.

```
The status of the creation for the patient. Either `succeeded` or `failed`. Depends on whether the respective demographics were accepted and used to create a Patient.
```

Reply:
Taken to Slack: https://metriport.slack.com/archives/C04DMKE9DME/p1738861571002049?thread_ts=1738614445.582769&cid=C04DMKE9DME